### PR TITLE
Multistore - Catalog > Stocks > Stocks & Movements - Update message in all shops or in a group context

### LINKED ISSUE #19457: Multistore - Catalog > Stocks > Stocks - Update message in all shops or in a group context
Currently, this page isn't working in all shops or in a group context
The current information message is "You can't manage your stock in this shop context: select a shop instead of a group of shops." 

For a better understanding, we should update the current message for the following one:
"Note that this page is available in a single shop context only. Switch context to work on it."

This is specified in the [multistore specials cases specs](https://github.com/PrestaShop/prestashop-specs/blob/master/broader-topics/multistorespecialsspecs.md#list-of-pages-for-specific-case-5-the-page-is-displayed-only-when-a-specific-shop-is-selected)

### LINKED ISSUE #19458: Multistore - Catalog > Stocks > Mouvements - Update message in all shops or in a group context
Currently, this page isn't working in all shops or in a group context
The current information message is "You can't manage your stock in this shop context: select a shop instead of a group of shops." 

For a better understanding, we should update the current message for the following one:
"Note that this page is available in a single shop context only. Switch context to work on it."

This is specified in the [multistore specials cases specs](https://github.com/PrestaShop/prestashop-specs/blob/master/broader-topics/multistorespecialsspecs.md#list-of-pages-for-specific-case-5-the-page-is-displayed-only-when-a-specific-shop-is-selected)

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Multistore - Catalog > Stocks > Stocks & Movements - Update message in all shops or in a group context
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #19457 & Fixes #19458
| How to test?      | Enable Multistore<br>Add a new shop<br>Go to `Catalog > Stocks > Stocks & Movements`<br>Check if the wording is changed

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27620)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Stock/overview.html.twig [hunk 1]
```diff
@@ -33,7 +33,7 @@
         <div id="stock-app"></div>
     {% else %}
         <div class="alert alert-danger" role="alert">
-            <p class="alert-text">{{ 'You can\'t manage your stock in this shop context: select a shop instead of a group of shops.'|trans({}, 'Admin.Catalog.Notification') }}</p>
+            <p class="alert-text">{{ 'Note that this page is available in a single shop context only. Switch context to work on it.'|trans({}, 'Admin.Catalog.Notification') }}</p>
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Stock/overview.html.twig]:
```suggestion
            <p class="alert-text">{{ 'Note that this page is available in a single shop context only. Switch context to work on it.'|trans({}, 'Admin.Notifications.Info') }}</p>
```
It'd be better to use the existing string. Thanks

Comment (Progi1984) [reply src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Stock/overview.html.twig]: Fixed

**Review by Julievrz — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]
Thanks @Progi1984

**Review by marwachelly — APPROVED** [review]
Hello, 

PR is tested. The wording is well changed. 
image
https://watch.screencastify.com/v/d4CxBTPg7Ki0uTAtyBdI

Thanks,

**Comment by Progi1984** [comment]
Thanks @marwachelly

**Pull request merged**